src/inject/settings/general.js

```diff
@@ -46,3 +46,17 @@ if (Settings.get('nativeFrame')) {
 } else {
   $('#native-frame').removeAttr('checked');
 }
+
+$('#scroll-lyrics').change((e) => {
+  Emitter.fire('settings:set', {
+    key: 'scrollLyrics',
+    value: $(e.currentTarget).is(':checked'),
+  });
+  Emitter.fireAtMain('settings:set:scrollLyrics', $(e.currentTarget).is(':checked'));
```

Comment:
This should be handled in the handler for `settings:set`.  See how we handle other events that require passthrough here.

https://github.com/MarshallOfSound/Google-Play-Music-Desktop-Player-UNOFFICIAL-/blob/master/src/main/features/core/desktopSettings.js#L67